spec.html

```diff
@@ -563,10 +563,10 @@ <h1><ins>CyclicModuleExecutionFulfilled ( _module_ )</ins></h1>
             1. Assert: _m_.[[DFSAncestorIndex]] is equal to _module_.[[DFSAncestorIndex]].
           1. Decrement _m_.[[PendingAsyncDependencies]] by 1.
           1. If _m_.[[PendingAsyncDependencies]] is 0 and _m_.[[EvaluationError]] is *undefined*, then
-            1. Assert _m_.[[Status]] is `"evaluating-async"`.
+            1. Assert: _m_.[[Status]] is `"evaluating-async"`.
             1. Let _cycleRoot_ be ! GetCycleRoot(_m_).
             1. If _cycleRoot_.[[EvaluationError]] is not *undefined*, return *undefined*.
-            1. Perform ! ExecuteCyclicModule(_module_).
+            1. Perform ! ExecuteCyclicModule(_m_).
```

Comment:
Unless this should be _cycleRoot_; I still don't quite understand the algorithm, but I'm pretty sure it's not supposed to be  _module_.

Reply:
The cycleRoot is used because, if you import a cycle and it's already run, then the error might be from something else in the cycle. Rather than propagating this back to each thing in the cycle, we look at the cycle root. I think this should be m as you wrote.